analyses/phylo_beta_diversity/phylo_beta_diversity.py

```diff
@@ -478,3 +498,194 @@ def calculate_phylo_beta_diversity_sorensen(pam, tree):
         Matrix(phylo_beta_sne_data, headers=mtx_headers),
         Matrix(beta_sor_data, headers=mtx_headers),
         Matrix(phylo_beta_sor_data, headers=mtx_headers))
+
+
+# .............................................................................
+def calc_phylo_jac_distr(pam, tree, obs, metric = "jac", nrand = 10):

```

Comment:
Rather than specify a single metric at a time to compute, I would rather compute the standard effect size for the entire family of computations.  That may not be straightforward without some additional tools though.  If you think you can do that easily, go for it.  If not, don't worry about it for now.